src/test/java/com/internship/socialnetwork/service/impl/UserServiceImplTest.java

```diff
@@ -0,0 +1,177 @@
+package com.internship.socialnetwork.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import com.internship.socialnetwork.exception.BadRequestException;
+import com.internship.socialnetwork.exception.NotFoundException;
+import org.junit.Before;
+import org.junit.Test;
```

Comment:
The test is not going to work since you are mixing JUnit4 and JUnit 5(jupiter) annotations. Pls run them before commit